build/images/base/Dockerfile

```diff
@@ -30,9 +30,15 @@ LABEL description="Takes care of building the Antrea binaries as part of buildin
 
 USER root
 
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    ipset \
-    jq \
- && rm -rf /var/lib/apt/lists/*
+# See https://github.com/kubernetes-sigs/iptables-wrappers
+# /iptables-wrapper-installer.sh will have permissions of 600.
+# --chmod=644 doesn't work with older versions of Docker and requires DOCKER_BUILDKIT=1, so we use
```

Comment:
Do you mean --chmod=755?

Reply:
I think I meant `700`, just need the owner to be able to execute the script. Let me fix it.